src/Core/src/Handlers/Label/LabelHandler.Standard.cs

```diff
@@ -11,5 +11,6 @@ public partial class LabelHandler : AbstractViewHandler<ILabel, object>
 		public static void MapFontFamily(LabelHandler handler, ILabel label) { }
 		public static void MapFontSize(LabelHandler handler, ILabel label) { }
 		public static void MapFontAttributes(LabelHandler handler, ILabel label) { }
+		public static void MapPadding(LabelHandler handler, ILabel label) {	}
```

Comment:
Add a mapping method to the Standard aspect of the handler; the Standard aspect is just a placeholder and is never actually called.